dbt_copilot_helper/templates/addons/env/aurora-postgres.yml

```diff
@@ -537,6 +537,6 @@ Resources:
     Properties:
       RoleArn: !Sub 'arn:aws:iam::${AWS::AccountId}:role/CWLtoSubscriptionFilterRole'
       LogGroupName: !Sub '/aws/rds/cluster/${{ '{' }}{{ addon_config.prefix }}DBCluster}/postgresql'
-      FilterName: !Sub '${App}-${Env}-stream'
+      FilterName: !Sub '/aws/rds/cluster/${App}/${Env}/${{ '{' }}{{ addon_config.prefix }}DBCluster}/postgresql'
```

Comment:
@lgarvey is it an issue if /app/ and /env/ aren't always in the same position in the name? Rds has more forward slashes here to specify cluster/instance

Reply:
It might be better to pull this fix into my PR that I will be raising shortly.

Reply:
This PR can be closed, I have pulled in these changes into my PR
https://github.com/uktrade/copilot-tools/pull/266/